packages/cysync-core/src/hooks/useAppUpdate.ts

```diff
@@ -107,7 +107,13 @@ export const useAppUpdate = () => {
     } finally {
       setIsUpdatesChecked(true);
     }
-  };
+  }, [
+    setIsUpdatesChecked,
+    setAppUpdateState,
+    autoUpdater.checkForUpdates,
+    setUpdateInfo,
+    onError,
```

Comment:
No need for dependencies here, since there are not state variables

Reply:
removed dependencies.